modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer/src/main/java/com/liferay/osb/site/initializer/OSBSiteInitializerImpl.java

```diff
@@ -74,6 +79,21 @@ public void addOrUpdateSXPBlueprint(
 		}
 	}
 
+	private String _replace(
```

Comment:
There are two private _replace methods. One in BundleSiteInitializer and this one in OSBSiteInitializerImpl.
Try to create a static method in SiteInitializerUtil to reuse it instead create another one.

Reply:
Done!